ext-apiserver/pkg/apis/cluster/v1alpha1/cluster_types_test.go

```diff
@@ -15,65 +14,68 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
 package v1alpha1_test
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	. "k8s.io/kube-deploy/ext-apiserver/pkg/apis/cluster/v1alpha1"
 	. "k8s.io/kube-deploy/ext-apiserver/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+	"reflect"
```

Comment:
It's a convention (I can't find a ref for it right now) that imports from the current project's code base are generally separate from imports that come from elsewhere. So `reflect` and `testing` should be in a separate block above the `metav1` import.